vignettes/Vaccines.Rmd

```diff
@@ -40,153 +123,262 @@ simparams <- get_parameters(list(
 
 simparams <- set_equilibrium(simparams, starting_EIR)
 
+# Run a model with no interventions in a setting with no seasonality
+output_control <- run_simulation(sim_length * 2, simparams)
 ```
 
-Then we can run the simulation for a variety of Vaccination strategies:
+Then we can run the simulation for a variety of vaccination strategies.
 
 ## Mass RTS,S
 
-This is a round of RTS,S vaccine for individuals between 5 months and 10 years followed by a booster after 18 months:
+First, we will model a single round of RTS,S vaccine given in a mass vaccination strategy where those within a given age range will be vaccinated to a specified coverage level and during a short window of time. In this example, individuals aged between 5 months and 10 years are vaccinated with a primary series (3 doses) followed by a single booster 12 months after the initial vaccination. The age groups to be vaccinated as well as timing of all vaccinations can be modified within the `set_mass_rtss()` function.
 
-```{r}
-rtssparams <- simparams
+We specify that the intervention should use the RTS,S vaccine through the `profile` and `booster_profile` inputs.
 
+```{r}
 rtssparams <- set_mass_rtss(
-  rtssparams,
-  timesteps = 1 * year,
-  coverages = 1,
-  min_wait = 0,
-  min_ages = 5 * month,
-  max_ages = 10 * year,
-  boosters = 18 * month,
-  booster_coverage = 0.95
+  simparams,
+  timesteps = 1 * year, # The single round of vaccination is at 1 year into the simulation.
+  coverages = 1, # The vaccine is given to 100% of the population between the specified ages.
+  min_wait = 0, # The minimum acceptable time since the last vaccination is 0.
```

Comment:
Add a note here for min_wait that the time is set to 0 because in our case we are only implementing one round of vaccination.